data/json/items/ammo/223.json

```diff
@@ -83,7 +83,7 @@
     "id": "556_mk318",
     "copy-from": "223",
     "type": "AMMO",
-    "name": { "str_sp": "5.56 NATO Mk318" },
+    "name": { "str_sp": "5.56 NATO Barrier" },
```

Comment:
What do these mean? If they're for intended purpose, eehh hard sell. M855 is already known for barrier penetration, calling this "Barrier" would make me think it's M855.